tests/framework/installer/ceph_installer.go

```diff
@@ -787,6 +778,13 @@ spec:
                       #
                       # ceph-volume setup also leaves ceph-UUID directories in /dev (just clutter)
                       rm -rf /dev/ceph-*
+                      # force wipe the disk
+                      set +Ee
+                      for rawdevice in $(lsblk -d --noheadings --list --output KNAME); do
+                        echo "attempting to zap /dev/$rawdevice"
+                        sgdisk --zap-all /dev/$rawdevice
```

Comment:
I don't understand why we need this, the `dd` above should have done the job already...

Also, this feels kinda dangerous for whoever tries to run the test somewhere else than the CI...
Even in the CI this could be troublesome since it's zapping the root drive:

```
+ echo 'attempting to zap /dev/xvda'
+ sgdisk --zap-all /dev/xvda
Creating new GPT entries.
Warning: The kernel is still using the old partition table.
The new table will be used at the next reboot or after you
run partprobe(8) or kpartx(8)
GPT data structures destroyed! You may now partition the disk using fdisk or
other utilities.
```

Also `-Z` **only** does:

```
Zap (destroy) the GPT and MBR data structures and then exit. 
This option works much like -z, but as it wipes the MBR as well as the GPT, 
it's more suitable if you want to repartition a disk after using this option, and completely unsuitable if you've already repartitioned the disk.
```

If we still want to use `sgdisk` I'd suggest hardcoding `/dev/xvdc`, just like the PV job does already.

Reply:
Ohhhh I get it, the PVC job still has bluestore labels because with Ceph 14.2.8 Rook uses raw mode so there is no LVM.
The only thing you need to do is to extract both `wipefs` and `dd` line outside of the PV loop and force `/dev/xvdc`.

Something like:

```bash
block=/dev/xvdc
wipefs --all "$block"
dd if=/dev/zero of="$block" bs=1M count=100 oflag=direct,dsync
```

Then `sgdisk` (and the new loop) is not needed.